Add Chilean banks icons

arthuraraujocl opened the pull request:
## Why are you adding this icons?
Shopify is missing icons for certain Chilean Banks, such as Banco Estado, Banco de Chile, Banco Falabella, BCI, Scotiabank, Itaú, Banco BICE, Banco Ripley, Banco Security, Banco Edwards, which need to be added for use as bank_transfer methods.

#### Help us identify yourself
- [X] I'm working/collaborating with the brand directly and they have provided the icons.
- [ ] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [X] All icons have a corresponding entry in `db/payment_icons.yml`
- [X] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [X] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [X] I am confident that all icons are clear and easy to read/understand
- [X] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [X] I have attached a screenshot comparison with the example icon provided in guidelines
- [X] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

## Attach a screenshot of the icon along side the example Visa icon
<img width="731" alt="image" src="https://github.com/user-attachments/assets/7db6fe0a-5459-4e9e-9111-8fa7d9316985" />


#### Tips how to create a screenshot
We have found free online SVG editor https://www.freecodeformat.com/svg-editor.php very useful to create one. Here is a sample code for you to verify that you icon appears properly along side the placeholder.

```
<!-- Change background color if needed to showcase your icon better -->
<style> body { background: black; } </style>

<!-- DO NOT DELETE EXAMPLE -->
<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-visa"><title id="pi-visa">Visa</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M28.3 10.1H28c-.4 1-.7 1.5-1 3h1.9c-.3-1.5-.3-2.2-.6-3zm2.9 5.9h-1.7c-.1 0-.1 0-.2-.1l-.2-.9-.1-.2h-2.4c-.1 0-.2 0-.2.2l-.3.9c0 .1-.1.1-.1.1h-2.1l.2-.5L27 8.7c0-.5.3-.7.8-.7h1.5c.1 0 .2 0 .2.2l1.4 6.5c.1.4.2.7.2 1.1.1.1.1.1.1.2zm-13.4-.3l.4-1.8c.1 0 .2.1.2.1.7.3 1.4.5 2.1.4.2 0 .5-.1.7-.2.5-.2.5-.7.1-1.1-.2-.2-.5-.3-.8-.5-.4-.2-.8-.4-1.1-.7-1.2-1-.8-2.4-.1-3.1.6-.4.9-.8 1.7-.8 1.2 0 2.5 0 3.1.2h.1c-.1.6-.2 1.1-.4 1.7-.5-.2-1-.4-1.5-.4-.3 0-.6 0-.9.1-.2 0-.3.1-.4.2-.2.2-.2.5 0 .7l.5.4c.4.2.8.4 1.1.6.5.3 1 .8 1.1 1.4.2.9-.1 1.7-.9 2.3-.5.4-.7.6-1.4.6-1.4 0-2.5.1-3.4-.2-.1.2-.1.2-.2.1zm-3.5.3c.1-.7.1-.7.2-1 .5-2.2 1-4.5 1.4-6.7.1-.2.1-.3.3-.3H18c-.2 1.2-.4 2.1-.7 3.2-.3 1.5-.6 3-1 4.5 0 .2-.1.2-.3.2M5 8.2c0-.1.2-.2.3-.2h3.4c.5 0 .9.3 1 .8l.9 4.4c0 .1 0 .1.1.2 0-.1.1-.1.1-.1l2.1-5.1c-.1-.1 0-.2.1-.2h2.1c0 .1 0 .1-.1.2l-3.1 7.3c-.1.2-.1.3-.2.4-.1.1-.3 0-.5 0H9.7c-.1 0-.2 0-.2-.2L7.9 9.5c-.2-.2-.5-.5-.9-.6-.6-.3-1.7-.5-1.9-.5L5 8.2z" fill="#142688"/></svg>

<!-- TODO: insert your icon here -->
<YOUR SVG CODE>

<br>
<!-- TODO: insert your icon here -->
<YOUR SVG CODE>
</br
```

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)

What's the expected date of this change to deploy on Shopify?

dannye0231 commented:
@arthuraraujocl are you able to check the following errors related to your SVGs?

Failure:
PaymentIconTest#test_Payment_icon_SVGs_are_a_single_line [test/unit/payment_icon_test.rb:127]:
The 'bancobice' SVG file should contain a single line of markup, optionally terminated by an empty line

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:124

.....F

Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [test/unit/payment_icon_test.rb:89]:
{:message=>"The 'paint0_linear_3326_81[20](https://github.com/activemerchant/payment_icons/actions/runs/13136944501/job/36654492687?pr=1292#step:4:21)' ID should be pi-bancobice-paint0_linear_3326_8120 (missing 'pi-' prefix)"}.
Expected /pi-(.*)/ to match "paint0_linear_3326_8120".

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

...

arthuraraujocl commented:
@dannye0231 SVGs files corrected. Please, let me know if it's OK. Thank you in advance.

arthuraraujocl commented:
Hi @dannye0231 @Lydia-shan-git , could you please update us on the status of this request? Thank you!

dannye0231 commented:
Hi @arthuraraujocl, are you able to check your SVG for the following issue?


Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [test/unit/payment_icon_test.rb:78]:
{:message=>"The 'itau' SVG file does not have the appropriate <title> value"}.
Expected: "Itau"
  Actual: "Itaú"

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

arthuraraujocl commented:
Hi @dannye0231. Fix done! All good to approve? Thank you.

arthuraraujocl commented:
Hi @Lydia-shan-git, please, when can we expect to have this PR deployed? Thank you!

claudiomunoz reviewed (changes requested):
Thank you for your submission, the icon file size for Itau and  BancoRipley are too large. Please run through SVGO and resubmit

arthuraraujocl commented:
Hi @claudiomunoz. The SVGO was making changes on the images. But I was able to compress both files now. Please, let me know if it's ok.

dannye0231 commented:
@arthuraraujocl can you please check your SVG for the following issue?

Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [test/unit/payment_icon_test.rb:68]:
{:message=>"The 'bancoripley' SVG file should have a single <title> tag"}.
Expected: 1
  Actual: 0

rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:61

arthuraraujocl commented:
Done @dannye0231. Please, let me know if it's ok now. Thank you!

dannye0231 commented:
@arthuraraujocl I found another issue come up:

Failure:
PaymentIconTest#test_Every_payment_icon_SVG_has_a_viewBox_attribute [test/unit/payment_icon_test.rb:56]:
{:message=>"The 'bancoripley' SVG file should have a 'viewBox' attribute on the root <svg> tag"}

bin/rails test /home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:53

...

arthuraraujocl commented:
@dannye0231 Fix done! And all checks made.

arthuraraujocl commented:
@claudiomunoz @johnchanshopify please, when can we expect to have this changes deployed? Thank you!

dannye0231 commented:
@arthuraraujocl are you able to re-base?  I am seeing some errors in the tests which is mostly likely something which will get resolved by re-basing your PR.

arthuraraujocl commented:
@dannye0231 please, let me know if it is good to go now. Thank you!

arthuraraujocl commented:
Hi @Lydia-shan-git , when can we expect for this changes to be approved? Thank you!

arthuraraujocl commented:
Hi @claudiomunoz @johnchanshopify @Lydia-shan-git  , when can we expect for this changes to be approved? Thank you!

dannye0231 commented:
@arthuraraujocl can you check something for me in itau.svg?

✖ The clip-path definition (url(#pi-clip-itau)) is referenced but not defined in the SVG. This could lead to rendering issues.

arthuraraujocl commented:
@dannye0231 Correction done in the itau.svg. Good to go now? Thank you!

arthuraraujocl commented:
Hi @dannye0231, I corrected the bancobice.svg and removed the bancoripley.svg, since I was unable to optimize it further more. Let me know if we are good to go now. Thank you!

dannye0231 commented:
Thanks, looks good now

dannye0231 reviewed (approved):
Criteria | Status | Notes
-- | -- | --
Format | ✔️ | SVG format is correct.
Dimensions | ✔️ | Width: 38, Height: 24.
Accessibility | ✔️ | Proper aria-labelledby usage.
Styles | ✔️ | Styles defined correctly.
Color Usage | ✔️ | Check against branding guidelines.
Content | ✔️ | Suitable for submission.
Opacity Requirement | ✔️ | Opacity set to 0.07; check design requirements.

arthuraraujocl commented:
Thank you @dannye0231!! When can we expect to see this changes applied?
